### ISSUE TITLE
Bugs when copying information from the attribution box

### ISSUE THREAD

**Issue by panchovm** [creativecommons/chooser#204]
## Description
The rich text tab within attribution box has the following bugs:
1. The links displayed are not copied correctly. It pastes a link mixed with the chooser link.
2. The license icons are not copied correctly. When trying in a text editor, it does not paste the images.

## Screenshots
<img width="546" alt="Screen Shot 2020-11-18 at 09 45 52" src="https://user-images.githubusercontent.com/895819/99532332-df133f80-2982-11eb-90f6-ec374ffd983a.png">

**Comment by Aspie96** [comment]
Actually I tried it and it copies it correctly.

The link is correct except for the issue in #210 and the icons are copied (although they are too small if pasted on OpenOffice Writer for some reason).

**Comment by Aspie96** [comment]
The reason for the size of the icons when pasted has to do with the "rem" unit of measurement.

If I change it to "em", writing in .attribution-tab .photo-license-icon "height: 1.4em" the icons appear of the exact same size as they currently do in the browser, but they become too large (instead of too small) in OpenOffice

**Comment by obulat** [comment]
The current version copies icons correctly to Word (I use WPS Office). Google Docs, however, shows image title (or alt text, I'm not sure) because it can't paste svg.

I've tinkered a little more with the copying of the rich text, and found a couple of things:
1. The best way of making icon sizes consistent I found was to actually set the width and height of the `img` inline, as that gets respected in word-processing software.
2. Google docs currently does not support pasting `svg` files as part of richtext. So, it is better to add license icons in `png` format and use them in richtext. HTML can still have svg links.
